changelogs/unreleased/5075-improve-exception-on-index-collision.yml

```
@@ -0,0 +1,6 @@
+description: Improve error reporting when an index collision is detected.
+issue-nr: 5075
+change-type: patch
+destination-branches: [master, iso5]
```

This will need to be updated of course.
```suggestion
destination-branches: [master, iso6]
```